prog/prio.go

```diff
@@ -104,7 +104,9 @@ func (target *Target) calcResourceUsage() map[string]map[int]weights {
 			case BufferFilename:
 				noteUsage(uses, c, 10, DirIn, "filename")
 			default:
-				panic("unknown buffer kind")
+				if bt := t.(*BufferType); !bt.IsCompressed() {
```

Comment:
Add `BufferCompressed` to the first case, it lists the types for which we don't introduce any priority relations.